Dockerfile

```diff
@@ -1,14 +1,23 @@
-FROM python:3.7-stretch
+FROM python:3.7
```

Comment:
I would explicitly specify the os using 3.7-buster. This makes sure we have a stable base os which changes only when we specify it in the image build.